doc/cortx-aws-k8s-installation.md

```diff
@@ -195,6 +195,33 @@ for ip in `aws ec2 describe-instances --filters Name=tag:Name,Values=$ClusterTag
 At this stage the Kubernetes cluster should be fully operational
 
 ## 3 Install CORTX 
+
```

Comment:
I'm not sure how this file got in here.  I didn't intend to make this change.

Reply:
This is fine. It's part of a previous merge that I did accepting a PR. So it's more about git process than anything. We can review in team call.